Microsoft.DotNet.Interactive/KernelExtensionLoader.cs

```diff
@@ -27,7 +27,7 @@ public async Task<bool> TryLoadFromAssembly(FileInfo assemblyFile, IKernel kerne
             var assembly = AssemblyLoadContext.Default.LoadFromAssemblyPath(assemblyFile.FullName);
             var extensionTypes = assembly
                                    .ExportedTypes
-                                   .Where(t => typeof(IKernelExtension).IsAssignableFrom(t))
+                                   .Where(t => typeof(IKernelExtension).IsAssignableFrom(t) && !t.IsInterface)
```

Comment:
There are other types that can't be instantiated: abstract classes and generic type definitions.

This same logic can be found here and could be refactored to an extension on `Type`, something like `CanBeInstantiated`:

https://github.com/dotnet/try/blob/f79ef5cc022b81e024eac68927eff83f9f25355d/Microsoft.DotNet.Interactive/CommandDeserializer.cs#L22-L25

Reply:
Done. Thanks